docker/.env

```diff
@@ -4,14 +4,14 @@ KEYCLOAK_SERVER_URL=https://fairspace-keycloak
 KEYCLOAK_REALM=fairspace
 KEYCLOAK_CLIENT_SECRET='**********'
 KEYCLOAK_CLIENT_ID=fairspace-client
-PLUTO_IMAGE=eu.gcr.io/fairspace-207108/sils-pluto:0.2.4
-SATURN_IMAGE=eu.gcr.io/fairspace-207108/sils-saturn:0.2.4
-FAIRSPACE_SSL_PROXY_IMAGE=eu.gcr.io/fairspace-207108/sils-fairspace-ssl-proxy:0.2.4
+PLUTO_IMAGE=eu.gcr.io/fairspace-207108/metatree-pluto:0.2.5
+SATURN_IMAGE=eu.gcr.io/fairspace-207108/metatree-saturn:0.2.5
+FAIRSPACE_SSL_PROXY_IMAGE=eu.gcr.io/fairspace-207108/metatree-fairspace-ssl-proxy:0.2.5
 #PLUTO_IMAGE=pluto-local:latest
 #SATURN_IMAGE=saturn-local:latest
 #FAIRSPACE_SSL_PROXY_IMAGE=fairspace-ssl-proxy-local:latest
 FAIRSPACE_HOSTNAME=fairspace
-FAIRSPACE_URL=https://fairspace
+METATREE_URL=https://fairspace
```

Comment:
I would change all or nothing. You changed Fairspace url, but not fairspace hostname... Maybe better to keep it as it was and change only the parts user can see, so documentation, footer, repo name?

Reply:
yes, FAIRSPACE_URL is heavily used in the documentation, but the others should be changed as well, now the .env is confusing.